### PR TITLE
Toggle Developer mode 

### PR DESCRIPTION
**Pull request by 0x0elliot**
I added the little change. I hope it's enough :>

### CODE REVIEW

File: ask.py [hunk 1]
```diff
@@ -698,6 +698,8 @@ def startup(file_name):
 	if is_dev:
 		print('\n')
 		pprint(tokens_list)
+                os.system('export FLASK_ENV=development')
+                #os.environ['FLASK_ENV']="development" #alternative cleaner method. i will let you decide which to keep.
```
Comment (Buscedv) [inline ask.py:702]: Good idea, use os.environ()

File: ask.py [hunk 2]
```diff
@@ -738,7 +740,7 @@ def startup(file_name):
 		# Stats the local development server.
```
Comment (Buscedv) [inline ask.py:740]: I noticed a spelling mistake made by me, but maybe you can fix it?

File: ask.py [hunk 3]
```diff
@@ -738,7 +740,7 @@ def startup(file_name):
...
 		# TODO: ALso support running the app in a production ready server.
 		style_print('Running Flask app:', styles=['bold'])
-		os.system('export FLASK_APP=app.py')
+                os.system('export FLASK_APP=app.py')
```
Comment (Buscedv) [inline ask.py:743]: Change this to os.environ()

**Review by Buscedv — CHANGES_REQUESTED** [review]
Thank you for your contribution! I have a few comments so check them and then I'll review this again 👍

File: ask.py (hunk 2 above)
Comment (0x0elliot) [reply ask.py:740]: on it!

**Comment by 0x0elliot** [comment]
i have fixed and made an extra little change. kindly review!

File: ask.py [hunk 4]
```diff
@@ -698,6 +698,7 @@ def startup(file_name):
 	if is_dev:
 		print('\n')
 		pprint(tokens_list)
+                os.environ['FLASK_ENV']='development'
```
Comment (Buscedv) [inline ask.py:701]: Remove the indent before `os.`. Put spaces around the equals sign.

File: ask.py [hunk 5]
```diff
@@ -735,10 +736,10 @@ def startup(file_name):
 			app.db.create_all()
 			print('\t✅')
 
-		# Stats the local development server.
+		# Starts the local development server.
 		# TODO: ALso support running the app in a production ready server.
 		style_print('Running Flask app:', styles=['bold'])
-		os.system('export FLASK_APP=app.py')
+                os.environ['FLASK_APP']='app.py'
```
Comment (Buscedv) [inline ask.py:742]: Remove the indent before `os.`. Put spaces around the equals sign.

**Review by Buscedv — CHANGES_REQUESTED** [review]
I noticed a few syntax problems. Also, make sure to use tabs and not spaces.

File: ask.py (hunk 5 above)
Comment (0x0elliot) [reply ask.py:742]: apologies for that. let me do that.

Comment (0x0elliot) [reply ask.py]: done. everything should be fine now

**Comment by 0x0elliot** [comment]
fixed all the errors and did a test run. everything seems to be running smooth now. i also understand the project better ^^ 
thanks for your patience. i appreciate it.

**Review by Buscedv — APPROVED** [review]
Thank you for your contribution!

**Pull request merged**